modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+= Deploying the Loki Collector
```

Comment:
@libander  Loki is not a collector.      We should use loki-operator, lokistack.